### PR TITLE
NET-1134:move oauth from CE build block to pro

### PR DESCRIPTION
**Pull request by yabinma**
## Checklist before requesting a review
- [x] My changes affect only 10 files or less.
- [x] I have performed a self-review of my code and tested it.
- [ ] If it is a new feature, I have added thorough tests, my code is <= 1450 lines.
- [ ] If it is a bugfix, my code is <= 200 lines.
- [ ] My functions are <= 80 lines.
- [ ] I have had my code reviewed by a peer.
- [ ] My unit tests pass locally.
- [x] Netmaker is awesome.

### CODE REVIEW

File: controllers/user.go [hunk 1]
```diff
@@ -8,12 +8,12 @@ import (
 
 	"github.com/gorilla/mux"
 	"github.com/gorilla/websocket"
-	"github.com/gravitl/netmaker/auth"
 	"github.com/gravitl/netmaker/database"
 	"github.com/gravitl/netmaker/logger"
 	"github.com/gravitl/netmaker/logic"
 	"github.com/gravitl/netmaker/models"
 	"github.com/gravitl/netmaker/mq"
+	"github.com/gravitl/netmaker/pro/auth"
```
Comment (abhishek9686) [inline controllers/user.go:16]: why is there a reference to pro pkg here?

File: controllers/enrollmentkeys.go [hunk 2]
```diff
@@ -10,11 +10,11 @@ import (
 	"github.com/google/uuid"
 	"github.com/gorilla/mux"
 
-	"github.com/gravitl/netmaker/auth"
 	"github.com/gravitl/netmaker/logger"
 	"github.com/gravitl/netmaker/logic"
 	"github.com/gravitl/netmaker/models"
 	"github.com/gravitl/netmaker/mq"
+	"github.com/gravitl/netmaker/pro/auth"
```
Comment (abhishek9686) [inline controllers/enrollmentkeys.go:17]: there shouldn't be any reference to pro from community

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

File: controllers/enrollmentkeys.go (hunk 2 above)
Comment (yabinma) [reply controllers/enrollmentkeys.go]: @abhishek9686 , I moved some of the common functions back to ce.  Please have a review again. Thanks.

**Review by abhishek9686 — APPROVED** [review]

**Comment by abhishek9686** [comment]
@yabinma resolve merge conflicts

**Pull request merged**